### DISCUSSION TITLE
Plot for a Particular PRN

### DISCUSSION THREAD

**Discussion by gnssLa** [groops-devs/groops#193] [Q&A] [answered]
Dear Groops Developers,

I am reaching out regarding the plotting for a single PRN using the method you presented in the [InstrumentGnssReceiver2TimeSeries](https://groops-devs.github.io/groops/html/InstrumentGnssReceiver2TimeSeries.html) documentation. I have been trying to plot the residuals for a single PRN using [PlotGraph.](https://groops-devs.github.io/groops/html/PlotGraph.html) Unfortunately, I have encountered some difficulties in implementing it successfully.

I have provided an example file for reference. 

[residuals_suth.zip](https://github.com/groops-devs/groops/files/11582602/residuals_suth.zip)

Could you please also guide me on how to plot the following:

- Elevation angle (e.g., A1*C**) for a specific PRN.
- STEC (I**C**).

NB: It is worth noting that this particular example pertains to BDS.

Thank you for your assistance.

Best regards,

Duarte.

**Comment by tmayerguerr** [comment, marked as answer]
Dear Duarte,

First of all, elevation is E1* and not azimuth A1* (in radian).
With [FunctionsCalculate](https://groops-devs.github.io/groops/html/FunctionsCalculate.html) you can remove the lines containing other PRNs. E.g. remove all except PRN 5 of BDS (67) with removalCriteria=`(data1 != 5) || (data2 != 67)`.
In my example [groops.zip](https://github.com/groops-devs/groops/files/11582823/groops.zip) I used additionally [InstrumentInsertNAN](https://groops-devs.github.io/groops/html/InstrumentInsertNAN.html) to break the plot lines at data gaps.

regards
Torsten
![resiudals](https://github.com/groops-devs/groops/assets/71824793/2c1a286e-1045-4201-8491-7dd07dcd9e44)

**Reply by gnssLa** [reply]
> Dear Duarte,
> 
> First of all, elevation is E1* and not azimuth A1* (in radian). With [FunctionsCalculate](https://groops-devs.github.io/groops/html/FunctionsCalculate.html) you can remove the lines containing other PRNs. E.g. remove all except PRN 5 of BDS (67) with removalCriteria=`(data1 != 5) || (data2 != 67)`. In my example [groops.zip](https://github.com/groops-devs/groops/files/11582823/groops.zip) I used additionally [InstrumentInsertNAN](https://groops-devs.github.io/groops/html/InstrumentInsertNAN.html) to break the plot lines at data gaps.
> 
> regards Torsten ![resiudals](https://user-images.githubusercontent.com/71824793/241426218-2c1a286e-1045-4201-8491-7dd07dcd9e44.png)

Dear @tmayerguerr,

Oops, I transposed the Elevation and Azimuth— that was a typo. Thanks for the correction. 

Thank you for being an exceptional educator and mentor.  I can now do what I wanted from GROOPS. 

Kind regards,

Duarte.
